clusterloader2/pkg/config/template_provider.go

```diff
@@ -161,8 +161,21 @@ func (tp *TemplateProvider) TemplateInto(path string, mapping map[string]interfa
 	return decodeInto(b, obj)
 }
 
-// GetOverridesMapping returns mapping from file specified by the given paths.
-func GetOverridesMapping(paths []string) (map[string]interface{}, error) {
+// LoadTestSuite creates test suite config from file specified by the given path.
+func LoadTestSuite(path string) (api.TestSuite, error) {
+	bin, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("test suite reading error: %v", err)
+	}
+	var testSuite api.TestSuite
+	if err = decodeInto(bin, &testSuite); err != nil {
+		return nil, err
+	}
+	return testSuite, nil
+}
+
+// LoadOverridesMapping returns mapping from file specified by the given paths.
+func LoadOverridesMapping(paths []string) (map[string]interface{}, error) {
```

Comment:
LoadTestOverrides  maybe?

[the word mapping doesn't appear anywhere in the api, flags, etc.]

Reply:
Done.